tests/unit/test_bruteforce.cpp

```diff
@@ -465,3 +465,32 @@ TEST_F(BruteForceTest, brute_force_search_empty_index) {
     VecSimQueryResult_Free(res);
     VecSimIndex_Free(index);
 }
+
+TEST_F(BruteForceTest, brute_force_test_RS_integration_test) {
+    VecSimParams params = {
```

Comment:
What does this test do? How does this scenario related to `inf` score?

Reply:
added comment